lib/motion/callback.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require "motion"
+
+module Motion
+  class Callback
+    attr_reader :broadcast
+
+    NAMESPACE = "motion:callback"
+    private_constant :NAMESPACE
+
+    def self.broadcast_for(component, method)
+      [
+        NAMESPACE,
+        component.stable_instance_identifier_for_callbacks,
```

Comment:
I want the name to be specific so that it isn't used for anything else. In previous versions of Motion, I have had a unique component identifier, but every time I have found a way to remove it and I think the code has been better for it being gone.

In this case, I think it is required, but until it is required for something else, I want to make its purpose clear and avoid using it for anything else so that it can be removed.